File: BackEnd/Models/servicio.cs
namespace BackEnd.Models
{
    public class servicio
    {
        public int idServicio { get; set; }
        public int idComercio { get; set; }

        public string nomServicio { get; set; }

        public string horaApertura { get; set; }

        public string horaCierre { get; set; }

        public int duracion { get; set; }

    }
}

File: src/app/servicio/servicio.ts

export interface servicioDTO{
    idServicio: number,
    idComercio: number,
    nomServicio : string,
    horaApertura: string,
    horaCierre: number,
    duracion: number
}

export interface servicioCreacionDTO{        
    idComercio: number,
    nomServicio : string,
    horaApertura: string,
    horaCierre: number,
    duracion: number
}